src/native/_mutable.js

```diff
@@ -194,13 +227,13 @@ module.exports = {
     mdata_entries_insert: Promisified(strToBuffer, []),
     mdata_entries_len: Promisified(null, t.usize),
     mdata_entries_get: Promisified(strToBuffer, [t.u8Pointer, t.usize, t.u64], readValueToBufferNoCapacity),
-    mdata_entries_for_each: Promisified(null, []),
+    mdata_entries_for_each: Promisified(callBackKeyValueLastEntry, []),
     mdata_entries_free: Promisified(null, []),
     mdata_keys_len: Promisified(null, t.usize),
-    mdata_keys_for_each: Promisified(null, []),
+    mdata_keys_for_each: Promisified(callBackLastEntry, []),
     mdata_keys_free: Promisified(null, []),
     mdata_values_len: Promisified(null, t.usize),
-    mdata_values_for_each: Promisified(null, []),
+    mdata_values_for_each: Promisified(callBackLastEntry, []),
```

Comment:
we need to do something more with the callback to wrap here. see comment before.